integration_tests/macros/unit_tests/test_get_test_min_and_max_buckets.sql

```diff
@@ -0,0 +1,276 @@
+{% macro test_get_test_buckets_min_and_max() %}
```

Comment:
There's no flow that I know of that runs these unit tests. Did you add one?

Reply:
TBH I was sure `run_unit_tests.py was part of the CI.

Reply:
I think the best way is to only add a pytest that runs this macro